librz/io/p/io_debug.c

```diff
@@ -340,15 +313,42 @@ static int fork_and_ptraceme_for_mac(RzIO *io, int bits, const char *cmd) {
 	posix_spawn_file_actions_destroy(&fileActions);
 	return p; // -1 ?
 }
-#endif // __APPLE__ && !__POWERPC__
 
-#if (!(__APPLE__ && !__POWERPC__))
+#else // USE_POSIX_SPAWN
+
 typedef struct fork_child_data_t {
 	RzIO *io;
 	int bits;
 	const char *cmd;
 } fork_child_data;
 
+#if __APPLE__ || __BSD__
+static void inferior_abort_handler(int pid) {
+	eprintf("Inferior received signal SIGABRT. Executing BKPT.\n");
```

Comment:
`RZ_LOG_ERROR()`?

Reply:
If I understand it right, this is not an error, it should be `RZ_LOG_INFO()`, but that is not printed by default (though I think it should, but that is another story), so I would keep it as eprintf until info is printed.

Reply:
use WARN then.

Reply:
It is not a warning either, it is just informative that the debugee got a SIGABRT.